dev-scripts/prepare-chart

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+function is_gnu_sed(){
+  sed --version >/dev/null 2>&1
+}
+
+function sed_i_wrapper(){
+  if is_gnu_sed; then
+    $(which sed) "$@"
+  else
+    a=()
+    for b in "$@"; do
+      [[ $b == '-i' ]] && a=("${a[@]}" "$b" "") || a=("${a[@]}" "$b")
+    done
+    $(which sed) "${a[@]}"
+  fi
+}
+
+
+DEV_SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+SCRIPT_DIR=$(realpath "$DEV_SCRIPT_DIR/../scripts")
+
+echo "Dev-Script: Prepare Rancher Chart"
+
+source "$SCRIPT_DIR/version"
+
+test_image="$REPO/rancher"
+
+echo "Preparing Rancher Chart for release: ${CHART_VERSION}"
+
+sed_i_wrapper -i -e "s/%VERSION%/${CHART_VERSION}/g" \
```

Comment:
I think you need separate commands for each of the files to work on macOS

Reply:
Yeah, reverted those being grouped together.